index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Portfolio for Matovu Farid Nkoba.">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/bac16a29bd.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header class="tool-bar">
+        <p>Welcome</p>
+        <i class="fa fa-bars" aria-hidden="true"></i>
+    </header>
+    <div class="background">
+        <main>
+            <section>
+                <header class="heading">
+                    <h1 class="blue-text">Hey there. I’m Farid</h1>
+                    <h1 class="green-text">I’m a software developer</h1>
+                </header>
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded, don’t hestiate to contact me.
+            </section>
+        </main>
+        <footer>
+            <div>
+                <i class="fa fa-twitter" aria-hidden="true"></i>
+                <i class="fa fa-linkedin" aria-hidden="true"></i>
+                <i class="fa fa-medium" aria-hidden="true"></i>
+                <i class="fa fa-github" aria-hidden="true"></i>
+                <i class="fa fa-angellist" aria-hidden="true"></i>
+            </div>
+        </footer>
+    </div>
```

Comment:
- [ ] By applying HTML semantic tags. I think it will be nice if we put this part of code in a `<section></section>` instead of a `<div>` tag. This will improve the skeleton of the HTML page.
- [ ] According to HTML semantic tag and HTML best practice in this project the text `I’m a software developer! I can help you build a product....`  is a paragraph `</p>`. Kindly put it in a `<p>` tag. I know you can do it :clap: :clap: 
- [ ] I think it would be nice to put the list of icons in a `<ul>` tag to group them and `<li>` per icon; each icon has a link `<a href=" ">` tag
- [ ] HTML heading tags are used to format headings on your page in order of importance. The `<h1>` tag is the one that comes first and shows search engines what content can be expected on the rest of the page. Great job :clap: :clap: I can see you put your name in a `<h1>`tag But kindly put the rest in a `<h2>`. For example : ` <h1>Your name</h1> </h2>I’m a software developer</h2>`.
- [ ] I know you have worked hard implementing this style however I think it would be nice if you can follow the same `paddings` and `margins` `colors` and `font-sizes` that are used in the template. Check here :point_down: 

| your design | the template|
| --------------- | ---------------- |
|![Screenshot from 2021-11-25 08-58-48](https://user-images.githubusercontent.com/65068771/143396635-4ade872b-e071-475d-9689-24e281dfdba7.png) | ![Screenshot from 2021-11-25 04-50-17](https://user-images.githubusercontent.com/65068771/143372900-e137a597-f2c1-4b38-9d43-fecae5729f2a.png) |




- [ ] A the bottom of your design I think you forgot to put the down button. Kindly let's add it as the template is designed. check :arrow_down: 
![Screenshot from 2021-11-25 04-59-57](https://user-images.githubusercontent.com/65068771/143374025-c7f5a956-8d6a-47ae-8250-e94e88773e78.png)